zebrad/src/commands.rs

```diff
@@ -21,6 +25,10 @@ pub const CONFIG_FILE: &str = "zebrad.toml";
 /// Zebrad Subcommands
 #[derive(Command, Debug, Options)]
 pub enum ZebradCmd {
+    /// The `copy-state` subcommand, used to debug cached chain state
+    #[options(help = "copy cached chain state (debug only)")]
+    CopyState(CopyStateCmd),
```

Comment:
I wonder if we can feature-gate commands like this, so it does not get compiled into `zebrad` by default

Reply:
There are a few different ways to implement this, and they depend on work we haven't done yet. So I opened ticket #3279 to make this change.